docs/docs/how_to/using-devcontainers.mdx

```diff
@@ -60,29 +60,15 @@ Github comes with a default codespace and you can use it to code your own devcon
 
 #### 3. Create a folder called `.devcontainer` in the root of your repository.
 
-#### 4. Create a Dockerfile in that folder, and paste the following code:
-
-```docker
-FROM --platform=linux/amd64 node:lts-bookworm-slim
-SHELL ["/bin/bash", "-c"]
-RUN apt update && apt install -y curl bash git tar gzip libc++-dev
-RUN curl -L https://raw.githubusercontent.com/noir-lang/noirup/main/install | bash
-ENV PATH="/root/.nargo/bin:$PATH"
-RUN noirup
-ENTRYPOINT ["nargo"]
-```
```

Comment:
Do we no longer need a Dockerfile?

Looks like you added one to simple_shield still: https://github.com/vezenovm/simple_shield/blob/9f11632ca3b06e3e8c639e45480263f4f89191f0/.devcontainer/Dockerfile